scripts/deploy.js

```diff
@@ -108,9 +124,10 @@ void (async () => {
       // Delete tar file.
       fs.unlinkSync(versionsTarFile);
     } catch (error) {
-      // If there was any error, stop. The initial directory should be uploaded manually.
-      console.log(error);
-      return;
+      console.error(
+        `Error downloading from HNS domain 'hnsDomain'. If this is a first-time deploy, please run with '--first-time'`
```

Comment:
Missing interpolation for `hnsDomain` :)

Reply:
```suggestion
        `Error downloading from HNS domain '${hnsDomain}'. If this is a first-time deploy, please run with '--first-time'`
```